bridge-stp.in

```diff
@@ -52,7 +58,7 @@ case $2 in
         if [ -e "$conf_file" ]; then
             . "$conf_file" || exit
         fi
-        checkpid $pid_file || exit 1
+        checkpid || exit 1
```

Comment:
a general question: isn't `/var/run` mounted as tmpfs, or doesn't it get cleaned up on reboot ?

it would seem that that would be a better solution to handle unclean reboots, than doing special pid handling

i agree there's the segfault case [ or some other mstpd crash type ]
in that case I'd also need to check how to do proper crash-handling